vendor/k8s.io/kubernetes/pkg/kubectl/resource/builder.go

```diff
@@ -341,6 +341,10 @@ func (b *Builder) LabelSelectorParam(s string) *Builder {
 // LabelSelector accepts a selector directly and will filter the resulting list by that object.
 // Use LabelSelectorParam instead for user input.
 func (b *Builder) LabelSelector(selector string) *Builder {
+	if len(selector) == 0 {
```

Comment:
what needed thsi.  It's the wrong fix.

Reply:
https://github.com/openshift/origin/blob/master/test/cmd/export.sh#L20 is failing with `error: selectors and the all flag cannot be used when passing resource/name arguments` since `cmd/export` [sets an empty selector param when one is not provided via the `--selector` flag](https://github.com/openshift/origin/blob/master/pkg/oc/cli/cmd/export.go#L126)